luda-editor/client/src/pages/sequence_edit_page/components/wysiwyg_editor/mover.rs

```diff
@@ -40,7 +41,7 @@ impl Component for Mover<'_> {
                         Color::TRANSPARENT,
                     )
                     .with_mouse_cursor({
-                        let is_dragging = matches!(dragging, Some(Dragging::Mover { .. }));
+                        let is_dragging = matches!(dragging, Some(Dragging::Mover { context }) if context.moving_with == MovingWith::Mouse);
```

Comment:
does it support fmt, even this line is very long?

Reply:
No, I don't think fmt is supported after adding `if context.moving_with == MovingWith::Mouse`.

Reply:
Fixed with 326feb3

Reply:
generally fmt doesn't work when the line is very long, especially when using macro